src/modules/powerrename/lib/PowerRenameItem.cpp

```diff
@@ -180,16 +179,6 @@ IFACEMETHODIMP CPowerRenameItem::GetId(_Out_ int* id)
     return S_OK;
 }
 
```

Comment:
Why was this removed?  Wouldn’t expect it in a rev for 1.1

Reply:
From the PR description. Looks like it isn't used and breaks the build.
> Removed PowerRename method was breaking the build because of the unresolved reference. Method is not needed, therefore deleted.